boards/adafruit-itsybitsy-m4/Kconfig

```diff
@@ -22,7 +22,12 @@ config BOARD_ADAFRUIT_ITSYBITSY_M4
     select HAS_PERIPH_TIMER
     select HAS_PERIPH_UART
     select HAS_PERIPH_USBDEV
+    select HAS_HIGHLEVEL_STDIO
+    select MODULE_BOARDS_COMMON_SAMDX1-ARDUINO-BOOTLOADER if MODULE_STDIO_CDC_ACM
```

Comment:
Probably we could instead:
```
config MODULE_BOARDS_COMMON_SAMDX1-ARDUINO-BOOTLOADER
   [...]
   default y if MODULE_STDIO_CDC_ACM
```
in `samdx1-arduino-bootloader/Kconfig`

Reply:
changed